x-pack/plugin/search-business-rules/src/main/java/org/elasticsearch/xpack/searchbusinessrules/PinnedQueryBuilder.java

```diff
@@ -105,7 +105,7 @@ private Item(String id) {
         /**
          * Read from a stream.
          */
-        Item(StreamInput in) throws IOException {
+        public Item(StreamInput in) throws IOException {
```

Comment:
Changing the visibility of this constructor is needed in order to use it in the ent-search module.

Reply:
Edit: And fields as well.